outwatch/src/main/scala/outwatch/dom/ChildCommand.scala

```diff
@@ -79,7 +79,7 @@ object ChildCommand {
           children.prepend(VNodeProxyNode(SnabbdomOps.toSnabbdom(node)))
         case ReplaceAll(list) =>
           children.clear()
-          children.push(list.map(node => VNodeProxyNode(SnabbdomOps.toSnabbdom(node))): _*)
+          children.push(list.map(node => VNodeProxyNode(SnabbdomOps.toSnabbdom(node))).toSeq: _*)
```

Comment:
Instead of mapping and converting, we can do `list.foreach(children.push(...))` directly instead.